Snippets/Parsing/ParseFile.swift

```diff
@@ -0,0 +1,13 @@
+//! Example on how to parse Markdown from file
+import Markdown
+import Foundation
+
+guard let url = Bundle.main.url(forResource: "Test", withExtension: "md") else {
```

Comment:
This is not working as expected like Swift Playground. What's the best practice for adding a Snippest-only resource file and how are we supposed to get the resource? 

Could you help provide some infomation on this? cc @bitjammer